runtime/doc/usr_08.txt

```diff
@@ -444,7 +444,8 @@ To go the other way use: >
 
 	[c
 
-Prepended a count to jump further away.
+Prepend a count to jump further away. Thus "4]c" jumps to the fourth next
+change, and "3[c" jumps to the third previous change.
```

Comment:
Shouldn't `4]c` and `3[c` have the same format as `[c` and others, i.e. on the separate lines?

Reply:
Yes, sorry, wasn't clear with the second comment -- it is fine as the same is used in other parts of the manual.